integration-tests/config/config.json

```diff
@@ -111,12 +115,9 @@
     {
       "path": "w3c/aria-practices",
       "url": "https://github.com/w3c/aria-practices.git",
-      "commit": "19c745ab9dd5e3e77d36afff848877c6b06c0114",
+      "commit": "2c38b7fa4a4c0de6daa15985dc44f8a29e92c60a",
       "args": [
-        "examples/**/*.html",
-        "aria-practices.html",
-        "**/*.md",
-        "*.md"
+        "**/*"
```

Comment:
It was picking up "zero files" with this glob
```suggestion
        "**/*.*"
```

Reply:
With `**/*` it seems to be picking up 336 files: [test (14.x, ubuntu-latest, w3c/aria-practices w3c/specberus)
](https://github.com/streetsidesoftware/cspell/runs/1611769763?check_suite_focus=true#step:5:58)

Reply:
Sorry, I did missread the snapshot, but the other pattern is what's used upstream